libraries/standard/http/src/private/http_client_internal.h

```diff
@@ -0,0 +1,85 @@
+/* Standard Includes. */
+#include <stdint.h>
+#include <stdbool.h>
```

Comment:
We shouldn't have a dependency on `stdbool.h` from our libraries as FreeRTOS codebase suggests against it (Some vendors are not C99 compliant)

Reply:
So do we use 1 or 0 in place of true or false?